nginx/run.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -ex
+
+K3S_MANIFEST_DIR=${K3S_MANIFEST_DIR:-/var/lib/rancher/k3s/server/manifests/}
+
+getConfig() {
+    local key=$1
+    _value=$(kairos-agent config get "${key} | @json" | tr -d '\n')
+    # Remove the quotes wrapping the value.
```

Comment:
```
host:/home/kairos # kairos-agent config get "nginx.version | @json" | tr -d 
INFO[2023-11-06T13:48:07Z] kairos-agent version v2.3.0                  
'"4.7.3"'
```
this is double quoted so the manipulation below removes just one set of quotes. That's probably the reason why it ends up double quoted above.